pkg/projects/util_test.go

```diff
@@ -1,18 +1,29 @@
 package projects
 
 import (
+	"context"
 	"fmt"
 	"github.com/go-resty/resty/v2"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"math"
 	"math/rand"
 	"net/http"
 	"reflect"
 	"strings"
 	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
+var testAccProviders = func() map[string]func() (*schema.Provider, error) {
```

Comment:
So, originally, when I wrote this, I used a closure so the `Provider` would basically be a singleton. But, since you have it working like this, you can remove the abstraction and just do this:

```go
func testAccountProvider() map[string]func() (*schema.Provider, error) {
        return map[string]func() (*schema.Provider, error){
		"project": func() (*schema.Provider, error) {
			return Provider(), nil
                  }
     }
}
```